packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -34,6 +34,7 @@ import {AgentContact, TASK_EVENTS} from '../../../src/services/task/types';
 import MetricsManager from '../../../src/metrics/MetricsManager';
 import { METRIC_EVENT_NAMES } from '../../../src/metrics/constants';
 import Mercury from '@webex/internal-plugin-mercury';
+import WebexRequest from '../../../src//services/core/WebexRequest';
```

Comment:
There is a typo in the path.
```suggestion
import WebexRequest from '../../../src/services/core/WebexRequest';
```

Reply:
Interesting to see the tests passing with this typo

Reply:
fixed